incoming_test.go

```diff
@@ -74,6 +74,23 @@ func TestPeersIncomingConnection(t *testing.T) {
 				"incoming connections should not be added to non-root peer list")
 		}
 
+		// verify number of peers/connections on the client side
+		opts := &IntrospectionOptions{}
+		serverState := ch.IntrospectState(opts).RootPeers
+		serverHostPort := ch.PeerInfo().HostPort
+
+		assert.Equal(t, len(serverState), 2, "Incorrect peer number")
```

Comment:
nit: in all the messages, replace number with count, "Incorrect peer count" "Incorrect outbound connection count" etc.